rhel7/profiles/stig-rhel7-disa.profile

```diff
@@ -215,8 +215,6 @@ selections:
     - audit_rules_file_deletion_events_unlinkat
     - rsyslog_remote_loghost
     - rsyslog_nolisten
-    - install_mcafee_antivirus
-    - mcafee_antivirus_definitions_updated
```

Comment:
@jamescassell why was this removed?

Reply:
The STIG no longer calls out mcafee in particular, but also allows e.g., clamav.

Reply:
@jamescassell please undo these changes as the STIG further more does call out McAfee.

Reply:
How would we update it to allow both mcafee and clamav?